src/message_interface/client_method.rs

```diff
@@ -157,24 +157,24 @@ pub enum ClientMethod {
         output_id: OutputId,
     },
     /// Get the milestone by the given milestone id.
-    GetMilestoneByMilestoneId {
+    GetMilestoneById {
         /// Milestone ID
         #[serde(rename = "milestoneId")]
         milestone_id: MilestoneId,
     },
     /// Get the milestone by the given index.
-    GetMilestoneByMilestoneIndex {
+    GetMilestoneByIndex {
         /// Milestone Index
         index: u32,
     },
     /// Get the UTXO changes by the given milestone id.
-    GetUtxoChangesByMilestoneId {
+    GetUtxoChangesById {
```

Comment:
`GetUtxoChangesById` doesn't tell which id;
maybe better to keep it like `GetUtxoChangesByMilestoneId` ?

Reply:
Idk, it's kind of obvious we're speaking of a milestone. We're putting a suffix to disambiguate id/index but at the end of the day we should have to put the parameter type name in the method name. Otherwise why not do `get_message_by_message_id` ? Because it's obvious

Reply:
good point, more or less obvious